cmd/buildah/bud.go

```diff
@@ -130,6 +134,17 @@ func budCmd(c *cli.Context) error {
 	if c.IsSet("file") || c.IsSet("f") {
 		dockerfiles = c.StringSlice("file")
 	}
+	format := "oci"
+	if c.IsSet("format") {
+		format = c.String("format")
```

Comment:
Nit, could call strings.ToLower() here, and make the code below look a little neeter.